admin_guide/available_ports.adoc

```diff
@@ -0,0 +1,117 @@
+= Available Ports
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+== Overview
+
+OpenShift infrastructure components talk to each other using ports;
+communication endpoints that are identifiable for specific processes or
+services. This topic is for finding which ports are required to be open between
+hosts, in the event that a firewall is being used between machines.
+
+.Node to Node
+[cols='2,8']
+|===
+| *4789*
+|Required for SDN communication between pods on separate hosts.
+|===
+
+.Nodes to Master
+[cols='2,8']
+|===
+| *53*
+|Required for SDN communication between pods on separate hosts.
+
+| *4789*
+|Required for SDN communication between pods on separate hosts.
+
+| *443* or *8443*
+|Required for the nodes to communicate to the master API, for the nodes to post back status, to receive tasks, et al..
+|===
+
+.Master to Node
+[cols='2,8']
+|===
+| *4789*
+|Required for SDN communication between pods on separate hosts.
+
+| *10250*
+|On Kubelet minions, the master proxies to nodes via the Kubelet for `oc`
```

Comment:
I take it this is a translation of "on the minons(Kublet)" from the KBase? Minions got renamed to nodes a while ago, so that should be the usage here, but I think you can safely just drop everything before the period instead.